### PR TITLE
 #755 modified Aliases.add to return a new instance

### PR DESCRIPTION
**Pull request by dmzaytsev**
for #755 

* modified `Aliases.add()` to return a new instance
* modified all implementation of `Aliases`

### CODE REVIEW

**Comment by karato** [comment]
@dmzaytsev Thanks, I will find someone to review this PR soon

**Comment by karato** [comment]
@pinaf review it

File: netbout-web/src/main/java/com/netbout/dynamo/DyAliases.java [hunk 1]
```diff
@@ -155,6 +155,7 @@ public void add(final String name) throws IOException {
                 .with(DyAliases.ATTR_LOCALE, Locale.ENGLISH)
         );
         Logger.info(this, "alias @%s added to %s", name, this.urn);
+        return new  DyAlias(this.region, item);
```
Comment (pinaf) [inline netbout-web/src/main/java/com/netbout/dynamo/DyAliases.java]: @dmzaytsev extra whitespace here

File: netbout-web/src/test/java/com/netbout/dynamo/DyAliasesITCase.java [hunk 2]
```diff
@@ -49,7 +49,7 @@ public void makesAlias() throws Exception {
         final Aliases aliases =
             new DyBase().user(new URN("urn:test:1")).aliases();
         final String name = "johnny";
-        aliases.add(name);
+        final Alias alias  = aliases.add(name);
```
Comment (pinaf) [inline netbout-web/src/test/java/com/netbout/dynamo/DyAliasesITCase.java]: @dmzaytsev exra whitespace here

**Comment by pinaf** [comment]
@dmzaytsev 2 minor comments

**Comment by dmzaytsev** [comment]
@pinaf fixed, please take a look

**Comment by pinaf** [comment]
@rultor merge

**Comment by karato** [comment]
@elenavolokhova review this ticket please, for compliance with our [quality rules](http://www.teamed.io/qa.html)

**Comment by karato** [comment]
@rultor please deploy

**Comment by elenavolokhova** [comment]
@pinaf 
According to our [quality rules](http://www.teamed.io/qa.html):
 > The code reviewer found at least three problems in the code.
 
 There were only two minor issues found during review.
Please confirm that you will try to find at least three problems next time.

**Comment by pinaf** [comment]
@elenavolokhova I confirm

**Comment by elenavolokhova** [comment]
@karato Quality is acceptable here.

**Comment by karato** [comment]
> @karato Quality is acceptable here.

@elenavolokhova thanks for the QA review, we'll work better next time

**Comment by karato** [comment]
@pinaf 10 mins was added to the account of @elenavolokhova (for QA review), in transaction 67924087. Many thanks! **20 mins** were added to your account in Transaction ID `AP-97V59274NV968862R` (task took 18 hours and 26 mins). review comments (c=5) added as a bonus. +20 added to your rating, current score is: [+4235](http://www.netbout.com/b/35128?open=rating)

**Pull request merged**